GVRf/Framework/src/org/gearvrf/GVRScene.java

```diff
@@ -17,34 +17,23 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 import org.gearvrf.GVRRenderData.GVRRenderMaskBit;
 import org.gearvrf.debug.GVRConsole;
-import org.gearvrf.script.IScriptable;
 import org.gearvrf.utility.Log;
 
-/**
- * The scene graph.
- *
- * It receives events defined in {@link ISceneEvents}. To add a listener to these events, use the
- * following code:
- * <pre>
- *     ISceneEvents mySceneEventListener = new ISceneEvents() {
- *         ...
- *     };
- *     getEventReceiver().addListener(mySceneEventListener);
- * </pre>
- */
-public class GVRScene extends GVRHybridObject implements PrettyPrint, IScriptable, IEventReceiver {
+/** The scene graph */
+public class GVRScene extends GVRHybridObject implements PrettyPrint, ISceneEvents {
     @SuppressWarnings("unused")
     private static final String TAG = Log.tag(GVRScene.class);
-
+    public static final Integer MAX_LIGHTS = 16;
     private final List<GVRSceneObject> mSceneObjects = new ArrayList<GVRSceneObject>();
     private GVRCameraRig mMainCameraRig;
     private StringBuilder mStatMessage = new StringBuilder();
-
-    private GVREventReceiver mEventReceiver = new GVREventReceiver(this);
```

Comment:
I think we lost code during the rebase; lines 47, 77, 29-38, 39, 461-501 - from what I can quickly tell. Or is it intentional?